specification/communication/Communication.Messages/models.tsp

```diff
@@ -142,7 +143,8 @@ model MessageTemplate {
 @discriminator("kind")
 @doc("The class describes a parameter of a template.")
 model MessageTemplateValue {
-  @doc("Name of the Template value")
+  @doc("Template binding reference name")
```

Comment:
Also this: is the description change asked by the azure reviewer?

Reply:
As explained in previous comments, I update this description to make it clear what this field means.